specs/_features/maxeb_increase/capella.py

```diff
@@ -1547,9 +1557,13 @@ def process_registry_updates(state: BeaconState) -> None:
         if is_eligible_for_activation(state, validator)
         # Order by the sequence of activation_eligibility_epoch setting and then index
     ], key=lambda index: (state.validators[index].activation_eligibility_epoch, index))
-    # Dequeued validators for activation up to churn limit
-    for index in activation_queue[:get_validator_churn_limit(state)]:
+    # Dequeue validators for activation up to churn limit [MODIFIED TO BE WEIGHT-SENSITIVE]
```

Comment:
consider removing ` [MODIFIED TO BE WEIGHT-SENSITIVE]`

Reply:
consider also keeping the typo `Dequeued` (vs Dequeue) to reduce the diff

Reply:
done! not sure `# Dequeued validators for activation up to churn limit` (the original) is even wrong tbh. will leave as is for now.